snapcraft/tests/test_options.py

```diff
@@ -21,44 +21,44 @@
 
 
 class OptionsTestCase(tests.TestCase):
-    @mock.patch('platform.machine', return_value='x86_64')
-    def test_amd64_support(self, mock_platform_machine):
-        try:
-            snapcraft.ProjectOptions()
-        except KeyError:
-            self.fail('Expected amd64 to be supported')
-
-    @mock.patch('platform.machine', return_value='i686')
-    def test_i386_support(self, mock_platform_machine):
-        try:
-            snapcraft.ProjectOptions()
-        except KeyError:
-            self.fail('Expected i386 to be supported')
-
-    @mock.patch('platform.machine', return_value='armv7l')
-    def test_arm_support(self, mock_platform_machine):
-        try:
-            snapcraft.ProjectOptions()
-        except KeyError:
-            self.fail('Expected arm to be supported')
 
-    @mock.patch('platform.machine', return_value='aarch64')
-    def test_arm64_support(self, mock_platform_machine):
-        try:
-            snapcraft.ProjectOptions()
-        except KeyError:
-            self.fail('Expected arm64 to be supported')
```

Comment:
I'm sorry @kyrofa, I couldn't deal with these tests that didn't have 100% coverage.